packages/outline-view/src/browser/outline-view-contribution.ts

```diff
@@ -36,6 +55,23 @@ export class OutlineViewContribution extends AbstractViewContribution<OutlineVie
         });
     }
 
+    /**
+     * COMMENT THIS METHOD TO DISABLE ADDING A TEST WIDGET !!!!!!!!!
+     */
+    @postConstruct()
+    init() {
+        this.applicationStateService.reachedState('ready').then(() => {
+            setTimeout(() => {
+                const widget = new TestWidget(this.messageService, this.frontendApplication, this.shellLayoutRestorer);
```

Comment:
you should create a widget via the factory and access here via `await this.widget`, please study `AbstractViewContribution`